### DISCUSSION TITLE
Modify Sanitization

### DISCUSSION THREAD

**Discussion by tonydero** [jmbannon/ytdl-sub#544] [Q&A] [answered]
Hello!

I'd like to be able to get a filename more similar to the output using `yt-dlp --restrict-filenames`  (i.e. no emoji, no spaces, etc.). Is there any way to customize the way the sanitization works to accomplish this, or perhaps even directly using the `--restrict-filenames` flag?

Thanks!

**Comment by jmbannon** [comment, marked as answer]
Use sanitized source variables, i.e. title_sanitized instead of title

**Reply by tonydero** [reply]
I am using the sanitized variables, and it is not removing spaces nor most emoji. I'm running version ``````````2023.03.15+7c692cd installed via pip.

**Reply by jmbannon** [reply]
I see, so a more strict sanitization. Currently there is no way, however they should be safe for use in both windows and unix file paths

**Reply by tonydero** [reply]
Understood, thanks.
